src/components/DetailViewHeader.jsx

```diff
@@ -57,6 +58,10 @@ const styles = (theme) => ({
     marginTop: '20px',
     marginBottom: '10px',
   },
+  viewSnapshotButton: {
+    width: '100%',
+    marginBottom: '12px',
+  },
 });
 
 export class DetailViewHeader extends React.PureComponent {
```

Comment:
Should this now be `OverviewHeader`?

Reply:
Yeah - I've done another pass now and renamed a bunch more files to corresponding data/overview names.